charts/lakefs/values.yaml

```diff
@@ -103,7 +103,7 @@ fluffy:
   enabled: false
   image:
     repository: treeverse/fluffy
-    tag: '0.2.3'
+    tag: '0.2.4'
```

Comment:
See the CI fail message - this file is part of the Chart package that means also need to bump the `version` in `Chart.yaml`